src/table/table.jsx

```diff
@@ -9,11 +10,25 @@ import TablePagination from '@material-ui/core/TablePagination';
 import TableRow from '@material-ui/core/TableRow';
 import TableBodyWrapper from './tableBody';
 
+const useStyles = makeStyles({
+  paper: {
+    height: '100%',
+  },
+  containerOverflowAuto: {
+    overflow: 'auto',
+  },
+  containerOverflowHidden: {
+    overflow: 'hidden',
+  },
+});
+
 export default function TableWrapper(props) {
-  const { tableData, setPageInfo } = props;
+  const { tableData, setPageInfo, constraints } = props;
   const { size, columns, rows } = tableData;
   const [page, setPage] = useState(0);
   const [rowsPerPage, setRowsPerPage] = useState(100);
+  const classes = useStyles();
+  const containerMode = constraints.active ? 'containerOverflowHidden' : 'containerOverflowAuto';
```

Comment:
This wasn't updating properly in. Since it changes I think it is better to keep it here. Could store it as a state, but I think it is a bit overkill